pilot/pkg/model/config_test.go

```diff
@@ -177,42 +178,70 @@ func TestServiceKey(t *testing.T) {
 func TestSubsetKey(t *testing.T) {
 	hostname := model.Hostname("hostname")
 	cases := []struct {
+		got      string
 		hostname model.Hostname
 		subset   string
 		port     *model.Port
 		want     string
+		err      error
 	}{
 		{
 			hostname: "hostname",
 			subset:   "subset",
 			port:     &model.Port{Name: "http", Port: 80, Protocol: model.ProtocolHTTP},
-			want:     "outbound|http|subset|hostname",
+			want:     "outbound.http.subset.hostname",
```

Comment:
Can we remove 'outbound' ? There is no value - no DNS name uses this pattern.

Also please add a "_" in front of port and subset.

Reply:
There is a lot of value in being able to distinguish traffic that is sent from client side vs traffic that actually reaches a particular instance. I don’t want to have a simple in.8080 as it’s not intuitive in terms of the hosts to which traffic is arriving at. And helps differentiate circuit breaker settings (inbound vs outbound for that service hostname).